web_ui/src/pages/project-details/components/project-models/models-container/model-card/model-card.component.tsx

```diff
@@ -111,7 +110,7 @@ export const ModelCard = ({
                     <Flex direction={'column'} width={'100%'} gap='size-100'>
                         <Flex alignItems={'center'} justifyContent={'space-between'}>
                             <Text UNSAFE_className={classes.modelInfo} data-testid={'trained-model-date-id'}>
-                                Trained: <Link to={modelUrl}>{formatDate(creationDate, 'DD MMM YYYY, hh:mm A')}</Link> |
+                                Trained: {formatDate(creationDate, 'DD MMM YYYY, hh:mm A')} |
```

Comment:
Can you elaborate this change? I see the `UnClickableLink` is being removed, but I'm kind of cofused why we had it in the first place. Does anyone know?